adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformVariants.scala

```diff
@@ -127,7 +130,7 @@ class TransformVariants(val args: TransformVariantsArgs)
       stringency = stringency)
 
     if (args.outputPath.endsWith(".vcf")) {
-      maybeSort(maybeCoalesce(variants.toVariantContexts)).saveAsVcf(args, stringency)
+      maybeSort(maybeCoalesce(variants).toVariantContexts).saveAsVcf(args, stringency)
```

Comment:
Note this change may have performance implications, I couldn't figure out how to maintain the generic signature for `maybeCoalesce`